app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [
+        Person(person.get("name"),
+               person.get("age"))
```

Comment:
doesn't it fit in 1 line in less than 79 characters?

Reply:
and if not, then parameters should be passed from the next lines after class name